docs/layer_util.js

```diff
@@ -382,6 +382,8 @@ function addLayerFromFeatures(layerDisplayData, index) {
         createStyleFunction(deckParams.colorFunctionProperties);
   }
   deckGlArray[index] = new deck.GeoJsonLayer({
+    // TODO: also add layer index to this id so we don't get duplicate ids from
```

Comment:
nit: unrelated? fine with leaving in, though

Reply:
Agreed, just thought of it during this PR.